sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -15,9 +16,9 @@ import { SASCredential } from '@azure/core-auth';
 
 // @public
 export interface AccessPolicy {
-    expiry: Date;
-    permission: string;
-    start: Date;
+    expiry?: Date;
```

Comment:
This is an intentional change. The service accepts empty access policies and Signed Identities without AccessPolicies. The Swagger misrepresented this, and that's the reason for the transform removing the required properties